base/util/src/main/java/org/apache/ofbiz/base/util/ScriptUtil.java

```diff
@@ -192,8 +192,9 @@ static ScriptContext createScriptContext(Map<String, Object> context) {
         localContext.put(WIDGET_CONTEXT_KEY, context);
         localContext.put("context", context);
         ScriptContext scriptContext = new SimpleScriptContext();
-        ScriptHelper helper = new ScriptHelperImpl(scriptContext);
-        localContext.put(SCRIPT_HELPER_KEY, helper);
+        //TODO: migration
```

Comment:
Here ScriptHelperImpl is from common package and pulls in a LOT of dependencies - from entity, service, etc. 
IMO that is not ok since entity, service are higher level than util.